packages/common/date/src/date.spec.ts

```diff
@@ -191,7 +191,7 @@ describe('getDateDistanceText', () => {
       ).toEqual('1시간 1분 1초');
     });
 
-    it(`'시간' 단위는 보여지지 않는다.`, () => {
+    it("When don`t wants show '시간' unit.", () => {
```

Comment:
`slash` use `it` keyword better than `test` keyword for method

`it` and `test` are functionally the same but semantically different.
```
it("should have a button", () => { ... }); // 1
test("should have a button", () => {}); // 2
```
You can read the code above as english sentence. Then which is the more easily readable? I think 1 is the winner.
```suggestion
    it("should not show '시간' unit", () => {
```

semantically in english

1. it should not show '시간' unit
2. it When don`t wants show '시간' unit.

I think 1 is more readibility🥳

ref: https://github.com/toss/slash/pull/37#discussion_r994630666

Reply:
Holy Cool man... i will fix `it`